sim/warlock/death_coil.go

```diff
@@ -17,6 +17,16 @@ func (warlock *Warlock) getDeathCoilBaseConfig(rank int) core.SpellConfig {
 
 	baseDamage *= 1 + warlock.shadowMasteryBonus()
 
+	healingSpell := warlock.GetOrRegisterSpell(core.SpellConfig{
+		ActionID:    core.ActionID{SpellID: spellId}.WithTag(1),
+		SpellSchool: core.SpellSchoolPhysical,
+		ProcMask:    core.ProcMaskSpellHealing,
+		Flags:       core.SpellFlagPassiveSpell,
```

Comment:
You may want to include `core.SpellFlagHelpful` here too

Reply:
added flag